model/custom_status.go

```diff
@@ -40,18 +42,28 @@ func (cs *CustomStatus) TrimMessage() {
 	}
 }
 
+func (cs *CustomStatus) SetDefaults() {
+	if cs.Emoji == "" {
+		cs.Emoji = DefaultCustomStatusEmoji
+	}
+
+	if cs.Duration == "" {
+		cs.Duration = DefaultCustomStatusDuration
+	}
+}
+
 func (cs *CustomStatus) ToJson() string {
 	csCopy := *cs
 	b, _ := json.Marshal(csCopy)
 	return string(b)
 }
 
 func (cs *CustomStatus) IsDurationValid() bool {
-	return ValidCustomStatusDuration[cs.Duration]
+	return cs.Duration == "" || validCustomStatusDuration[cs.Duration]
 }
 
 func (cs *CustomStatus) IsExpirationTimeValid() bool {
-	return !(cs.Duration != "dont_clear" && (cs.ExpiresAt.IsZero() || cs.ExpiresAt.Before(time.Now())))
+	return !(cs.Duration != "" && cs.Duration != "dont_clear" && (cs.ExpiresAt.IsZero() || cs.ExpiresAt.Before(time.Now())))
```

Comment:
nit - maybe refactor it like this 

```suggestion
	isStatusExpiryEmpty := cs.Duration == "" || cs.Duration == "dont_clear"
	isStatusAlreadyExpired := !cs.ExpiresAt.IsZero() &&  cs.ExpiresAt.Before(time.Now())
	return isStatusExpiryEmpty || !isStatusAlreadyExpired
```